Makefile

```diff
@@ -22,8 +22,7 @@ GOOSES   ?= "linux windows" # To override at the cli do: GOOSES="\"darwin bsd\""
 GOARCHES ?= "amd64 arm64" # To override at the cli do: GOARCHES="\"ppc64 mips\""
 ltsc2019  = "10.0.17763.4010"
 ltsc2022  = "10.0.20348.643"
-ltsc2022-KB5035857   = "10.0.26100.1"
-win2025 = "10.0.26100.1"
+win2025  = "10.0.26244.5000"
```

Comment:
Lets go ahead and change this to `ltsc2025` and make the appropriate changes in the pipeline and docker files.

Reply:
version used for win2025 is not officially GA'ed yet. Should we "ltsc" prefix when it is not GA'ed?

Reply:
Hi John, I think we can use win2025 or similar name until win2025 is officially released.

Reply:
if you call it `ltsc2025` now, you only need to update the value of this variable in this once place when it is released for real. 
if you call it `win2025` now, when it is released you will need to update value AND the variable name everywhere.
the intent of this work is to support ltsc2025, right? call it that.

Reply:
and fix the alignment